extractors/makeTheFlows_DZ_and_IZ.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+import sys
+
+upward = '00462936.csv'
+flows = 'wu03buk_oa_wz_v4.csv'
```

Comment:
Is that publicly available? If it is, it'd be nice if the script would download it, otherwise, a reference to the data source would be nice either way.